deepface/DeepFace.py

```diff
@@ -702,23 +716,23 @@ def represent(
             align=align,
         )
     else:  # skip
-        if isinstance(img_path, str):
-            img = functions.load_image(img_path)
-        elif type(img_path).__module__ == np.__name__:
+        if type(img_path).__module__ == np.__name__:
```

Comment:
I think we do not need to check type. load_image handles loading different type of inputs. So, we can get rid of the if and else block here.

Reply:
Previously I find the diffrence between here and `load_image` for copy. But after carefully look, no copy is ok. I'll do it.